tests/integration/organizations_test.go

```diff
@@ -99,6 +99,30 @@ func TestOrganizations(t *testing.T) {
 		assert.NotEmpty(t, organizationMembership.ID)
 	}
 
+	createdOrganizationMembership, err := client.Organizations().CreateMembership(newOrganization.ID, clerk.CreateOrganizationMembershipParams{
+		UserID: users[1].ID,
+		Role:   "admin",
+	})
+	if err != nil {
+		t.Fatalf("Organizations.CreateMembership returned error: %v", err)
+	}
+	if createdOrganizationMembership == nil {
+		t.Fatalf("Organizations.CreateMembership returned nil")
+	}
+	assert.Equal(t, createdOrganizationMembership.Role, "admin")
+
+	updatedOrganizationMembership, err := client.Organizations().UpdateMembership(newOrganization.ID, clerk.UpdateOrganizationMembershipParams{
+		UserID: organizationMemberships.Data[0].PublicUserData.UserID,
+		Role:   "guest_member",
```

Comment:
```suggestion
		Role:   "basic_member",
```
Let's change this to basic_member, since guest_member is not supported.